packages/design-system/src/scalars/components/amount-field/amount-field.stories.tsx

```diff
@@ -176,6 +176,22 @@ export default meta;
 type Story = StoryObj<typeof meta>;
 
 export const Default: Story = {
+  args: {
+    placeholder: "0",
+    label: "Enter Amount and Select Currency",
+    name: "amount",
+    step: 0,
+    type: "AmountCurrencyFiat",
+    allowedCurrencies: ["USD", "EUR"],
+    currencyPosition: "right",
+    value: {
+      amount: undefined,
+      currency: "USD",
+    },
+  },
```

Comment:
Why do we need `step` and `currencyPosition` here? This is the default variant so it should be minimal args to represent the default usage of the component

Reply:
done